pepper-apis/docs/specification/src/endpoints/user.studies.activities.yml

```diff
@@ -31,7 +31,16 @@ post:
   tags:
     - Activities
   summary: create a new instance of an activity
-  description: Given an `activityCode`, the server will create a new instance of the activity for the specified user if the configured conditions for the activity is met.
```

Comment:
@alextanlan @bskinner I've updated documentation descriptions to better describe parent/child dependency when creating child instances, particularly about read-only behavior.

Reply:
Thank you for the heads up